packages/components/src/modal/index.tsx

```diff
@@ -47,7 +55,7 @@ function UnforwardedModal(
 		bodyOpenClassName = 'modal-open',
 		role = 'dialog',
 		title = null,
-		focusOnMount = true,
+		focusOnMount = focusFirstNonCloseButtonElement,
```

Comment:
Can we avoid supporting a function in the "Modal" component? Just make "true" work like that basically?

Reply:
> If this property is true, it will focus the first tabbable element rendered in the modal.

I think we should and can do this. I'll update now.

Reply:
✅ in https://github.com/WordPress/gutenberg/pull/54296/commits/1f71011985d4822cb24c934802412c06a102f65e.

Now Modal interface remains the same but behaviour changes under the hood.